src/arduino/arduino.ts

```diff
@@ -725,9 +725,14 @@ export class ArduinoApp {
             }
             return ret;
         }
+        let stdoutbuf = ""
         const stdoutcb = (line: string) => {

```

Comment:
The name `line` is obviously misleading since it really is chunks of stdout text, not specifically split into lines, so this name should be updated as well.

I suggest `stdoutText` which is descriptive and also matches the documentation to the `spawn` function in src/common/util.ts (which is missing an update to match its current parameters, great if you could fix that as well).

With a rename of the function argument in `stdoutcb` (and fix `stderrcb` as well at the same time), then you could use `stdoutText` directly without needing to copy to a `stdoutbuf` variable.